implementation/wp-includes/fields-api/control-types/class-wp-fields-api-wysiwyg-control.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * @package    WordPress
+ * @subpackage Fields_API
+ */
+
+/**
+ * Fields API Textarea Control class.
+ *
+ * @see WP_Fields_API_Control
+ */
+class WP_Fields_API_WYSIWYG_Control extends WP_Fields_API_Control {
+
+	/**
+	 * {@inheritdoc}
+	 */
+	public $type = 'wysiwyg';
+
+	/**
+	 * {@inheritdoc}
+	 */
+	protected function render_content() {
+		
+		$this->input_attrs();
+		wp_editor( esc_html( $this->value() ), $this->input_attrs['id'] );
+		echo '<pre>', var_dump( $this->input_attrs ), "\n", $this->id, "\n", $this->value, '</pre>';
+		spew_var($this);
+
+	}
+
+	/**
+	 * {@inheritdoc}
+	 */
+	public function render_attrs() {
```

Comment:
You can remove this now (pull latest from fields api `develop` branch over to yours to get the changes.